src/main/java/com/google/sps/model/Trip.java

```diff
@@ -3,17 +3,9 @@
 import java.util.List;
 
 public class Trip {
-    private long tripID;
-    private String title;
-    private List<Long> participants;
+    public long tripID;
+    public String title;
+    public List<Long> participants;
     public float totalCost;
-    private float totalBudget;
-    
-    public long getParticipantAmmountOwed(long userId){
-        //TODO: calculate participant ammount owed
-    }
-
-    public long getParticipantAmmountPaid(long userId){
-        //TODO: calculate ammount paid  by a user
-    }
+    public float totalBudget;
```

Comment:
ok to leave it public. 
Just FYI: for value classes like this, you would normally make the class variables private and have get & set methods to access the data. If you are interested you can look into AutoValue classes. That is an easy way to build data classes like this - https://chromium.googlesource.com/external/github.com/google/auto/+/auto-value-1.6.2/value/userguide/index.md